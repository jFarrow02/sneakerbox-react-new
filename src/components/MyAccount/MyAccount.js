import React from 'react';
import './MyAccount.scss';

const MyAccount = (props)=>{
 
    return(
        <section className='MyAccount'>
            My Account here
        </section>
    );
};

export default MyAccount;