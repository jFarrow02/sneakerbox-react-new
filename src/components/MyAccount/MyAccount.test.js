import React from 'react';
import MyAccount from './MyAccount';
import renderer from 'react-test-renderer';

it('should match a snapshot', ()=>{
    const tree = renderer.create(<MyAccount/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
})