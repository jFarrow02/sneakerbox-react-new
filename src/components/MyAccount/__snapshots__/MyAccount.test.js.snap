// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match a snapshot 1`] = `
<section
  className="MyAccount"
>
  My Account here
</section>
`;
